CHANGELOG.md

```diff
@@ -8,9 +8,12 @@
 - Tests - Add MySQL min (5.6) and max (8.0) versions.
 - Database Support - Add support for MySQL 8.
 - Resource Deletion - Users can now bulk delete records.
+- Technical - Add test and documentation for query-builder sort method.
+- Smart Actions - Add a method to `RecordsGetter` to get all models IDs given a query or an ID list.
```

Comment:
Why did the 6.0.0-beta.0 release disappeared in the changelog?

Reply:
Because there is nothing else in release 6.0.0 so it would have been empty. I thought (since it was never merged on devel) that it should then have dissapeared (see: https://forestadmin.slack.com/archives/GAZF5Q5RV/p1584454701034000)